### DISCUSSION TITLE
doris 2.0.0 can not export table to s3

### DISCUSSION THREAD

**Discussion by lymaimei** [apache/doris#22055] [A - General / Q&A] [answered]
![image](https://github.com/apache/doris/assets/19678288/d652e341-3292-4614-b5c4-a886ef68ad00)
![image](https://github.com/apache/doris/assets/19678288/2720c9a2-631e-4f43-8d86-0fbdf6dfeae1)

**Comment by LemonLiTree** [comment, marked as answer]
https://doris.apache.org/docs/dev/data-operate/export/outfile/
try outfile

**Comment by lymaimei** [comment]
end point should be http:// not https://
case closed
